Autopilot/Path Manager/PathManager.c

```diff
@@ -667,12 +667,37 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+// check if power on reset, brown out reset, watchdog time out or software reset flags are set
+
+ char checkForReset(){
+     if (RCONbits.POR || RCONbits.BOR || RCONbits.WDTO || RCONbits.SWR || RCONbits.EXTR){
```

Comment:
Comment on the registers you're checking. Remember that if a watchdog reset or software reset occured, we DO NOT want to reset the home coordinates. The only time we'll want it reset is if it was due to a power or hardware reset.